lblod/frontend-lokaal-mandatenbeheer
# LMB-388: Tabullar component for draft mandataris instances in orgaan

## Description

The current table in the `orgaan/mandatarissen` route looks like this.
![image](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/36266973/de5384a5-0446-4492-a607-df0b634ec78f)

We created a new component because we want that the user is able to edit the draft mandatarissen. We add configuration for showing columns. This because we want to use this component multiple times but not with all columns always visible.
All links to persons or mandatarissen are also removed. 
![image](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/36266973/3b005dfa-2864-4ea6-9644-ac4fe750b491)


## How to test

1. Go to the corresponding link http://localhost:4200/organen/<ID>/mandatarissen and see if the tables contains all the columns as before (voornaam, achternaam, fracties, mandaat, start- en eind datum)
2. You are not routed to the person or mandaat when clicking on it
3. You do not have pagination
